docs/whats-new.md

```diff
@@ -152,25 +176,6 @@ This release adds a number of new geospatial format loaders
 
 ### New Geospatial Loaders
 
-<table style="border: 0;" align="center">
-  <tbody>
-    <tr>
-      <td>
-        <img style="max-height:200px" src="https://raw.github.com/visgl/deck.gl-data/master/images/whats-new/mvt-layer.jpg" />
-        <p><i>MVTLoader + <a target="_blank" href="https://deck.gl/#/documentation/deckgl-api-reference/layers/mvt-layer">MVTLayer</a></i></p>
-      </td>
-      <td>
-        <img style="max-height:200px" src="https://raw.github.com/visgl/deck.gl-data/master/images/whats-new/terrain.jpg" />
-        <p><i>TerrainLoader + <a target="_blank" href="https://deck.gl/#/documentation/deckgl-api-reference/layers/terrain-layer">TerrainLayer</a></i></p>
-      </td>
-      <td>
-        <img style="max-height:200px" src="https://raw.github.com/visgl/deck.gl-data/master/images/whats-new/esri-i3s.gif" />
-        <p><i>I3SLoader + <a target="_blank" href="https://deck.gl/#/documentation/deckgl-api-reference/layers/tile-3d-layer">Tile3DLayer</a> + I3S</i></p>
-      </td>
-    </tr>
-  </tbody>
-</table>
-
```

Comment:
Is this deletion intentional?

Reply:
Hmm... Looks like I just moved it to the top to have the pretty images front and center

Reply:
I see, but those images are specific to what was new in 2.1, no?

Reply:
Yes but images help generate excitement about frameworks, so still worth keeping them up top :) I assume most people don't check this page on a frequent basis. 

But you have a point, I will add versions to the comments in follow-up PR.